example/package.json

```diff
@@ -10,9 +10,13 @@
   "license": "ISC",
   "devDependencies": {
     "@babel/core": "^7.7.5",
-    "@storybook/addons": "^5.2.8",
-    "@storybook/react": "^5.2.8",
+    "@storybook/addons": "^5.3.13",
+    "@storybook/preset-create-react-app": "^1.5.2",
+    "@storybook/react": "^5.3.13",
     "babel-loader": "^8.0.6",
-    "storybook-addon-styled-component-theme": "link:../"
+    "storybook-addon-styled-component-theme": "1.3.0",
```

Comment:
please keep `link:../`

Reply:
Just out of curiosity does this make it point to current version in parent folder? 👀 Didnt seem to want to run out of the box with npm